packages/node/src/config.js

```diff
@@ -4,6 +4,10 @@ const os = require('os')
 const { inspect } = require('util')
 
 module.exports = {
+  appType: {
+    ...schema.appType,
+    defaultValue: () => 'node'
```

Comment:
Should we update the Node MazeRunner tests to check for this too?

Reply:
I did… but it turns out I didn't add it to the commit :)